msal/src/main/java/com/microsoft/identity/client/internal/MsalUtils.java

```diff
@@ -234,8 +237,26 @@ public static int getExpiryOrDefault(String expiresIn) {
      * @param url
      * @return
      */
+    @Deprecated
     public static boolean hasCustomTabRedirectActivity(@NonNull final Context context,
                                                        @NonNull final String url) {
+        try {
+            return validateCustomTabRedirectActivity(context, url);
+        } catch (final MsalClientException e) {
+            return false;
+        }
+    }
+
+    /**
+     * validateCustomTabRedirectActivity - Ensures that the developer has properly configured their
+     * AndroidManifest to expose the BrowserTabActivity.
+     *
+     * @param context
+     * @param url
```

Comment:
nit: javadoc, or if it isn't meant to be public, no javadoc.

Reply:
Addressed